streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -194,7 +194,7 @@ private void logRestoreOffsets(final TopicPartition partition,
     private Collection<TopicPartition> completed() {
         final Set<TopicPartition> completed = new HashSet<>(stateRestorers.keySet());
         completed.removeAll(needsRestoring.keySet());
-        log.debug("completed partitions {}", completed);
+        log.trace("Restoring completed partitions so far: {}", completed);
```

Comment:
The wording here is a bit confusing. It makes it sound like we are restoring partitions that have already been completed

Reply:
maybe "Restoration completed for partitions:"

Reply:
what though me off with the original message is that it gave the impression that it had just finished restoring those partitions, giving the impression it kept restoring them over and over again. In reality this log message just reports that restoration is still in progress and that so far we have completed the given set of partitions, but it might log this message at arbitrary times during the restoration process, not only when a partition was completed.

Reply:
Updated the message a bit; I'm also fine with removing this log4j entry if people prefer this.